test/jdk/java/net/httpclient/Response1xxTest.java

```diff
@@ -102,6 +127,10 @@ public void teardown() throws Exception {
             http2Server.stop();
             System.out.println("Stopped HTTP2 server");
         }
+        if (https2Server != null) {
+            https2Server.stop();
+            System.out.println("Stopped (https) HTTP2 server");
+        }
```

Comment:
This test should probably use a `final ReferenceTracker TRACKER = ReferenceTracker.INSTANCE;` now to double check that all clients have properly terminated without any dangling operations at the end of the test.

You can `grep TRACKER` in existing HttpClient tests to see how this is used.
The TRACKER can be used to track created clients, and invoked at the end of the test to verify that all resources have been properly reclaimed.